recipe/meta.yaml

```diff
@@ -8,8 +8,8 @@ source:
   url: http://ftp.gnu.org/gnu/gsl/gsl-{{ version }}.tar.gz
   sha256: dcb0fbd43048832b757ff9942691a8dd70026d5da0ff85601e52687f6deeb34b
   patches:
-    # diff from source distribution to https://github.com/ampl/gsl/tree/v2.4.0
-    - patch_cmake_for_windows.diff  # [win]
+    # diff from source distribution to https://github.com/ampl/gsl/tree/v2.7.0
+    - 0001-patch_cmake_for_windows.diff  # [win]
```

Comment:
please add to 'build-requirements' m2-patch dependency for Windows

Reply:
Thanks! Added this to the build requirements: [see](https://github.com/AnacondaRecipes/gsl-feedstock/pull/3/commits/6641ec3c31bb203c9a2be3896bf5d98a3133a954)